reference/7.4/PSReadLine/Get-PSReadLineOption.md

```diff
@@ -44,35 +44,43 @@ HistorySaveStyle                       : SaveIncrementally
 HistorySearchCaseSensitive             : False
 HistorySearchCursorMovesToEnd          : False
 MaximumHistoryCount                    : 4096
-ContinuationPrompt                     : >>
+ContinuationPrompt                     :
```

Comment:
It shows `>>` to me by default

Reply:
My bad for not running this under `-noprofile`.